NorthstarDedicatedTest/dedicated.cpp

```diff
@@ -265,6 +265,8 @@ void InitialiseDedicated(HMODULE engineAddress)
 	// make sure it still gets registered
 
 	// add cmdline args that are good for dedi
+	CommandLine()->AppendParm("+fatal_script_errors 0", 0);
+	CommandLine()->AppendParm("-safe", 0);
```

Comment:
```suggestion
	CommandLine()->AppendParm("-safe", 0);
```
Let's change it to just `-safe` and we can give it a better look but as Bob said, it might not be necessary either to use.